gorm-tools/src/main/groovy/yakworks/gorm/boot/GormToolsConfiguration.groovy

```diff
@@ -109,8 +111,8 @@ class GormToolsConfiguration {
     TrxService trxService(){ new TrxService()}
 
     @Bean
-    GormToolsTrxManagerBeanPostProcessor gormToolsTrxManagerBeanPostProcessor() {
-        return new GormToolsTrxManagerBeanPostProcessor()
+    GormToolsPreQueryEventListener gormToolsPreQueryEventListener() {
```

Comment:
I thought we were going to use 2 event listeners. one in rally for security and one in main gorm-tools.

Where is the Max stuff?

Reply:
Done